src/components/Dropdown/ItemContainer.tsx

```diff
@@ -31,7 +32,8 @@ export const boxStyles = classnames(
   padding('p-3'),
   maxHeight('max-h-64'),
   overflow('overflow-y-auto'),
-  alignItems('items-start')
+  alignItems('items-start'),
+  width('w-max')
```

Comment:
Won't it lead to the problem with dropdowns in other places, like simple dropdown?